examples/ComplexExample/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "RNExperimental",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "start": "node node_modules/react-native/local-cli/cli.js start",
+    "test": "jest"
+  },
+  "dependencies": {
+    "react": "15.4.2",
+    "react-native": "0.40.0",
+    "react-native-vector-icons": "^4.0.0",
+    "react-navigation": "git+https://github.com/react-community/react-navigation.git",
```

Comment:
How well does this work? The other examples are set up in such a way that they can be developed alongside react-navigation. It doesn't look like that can happen here.